chapters/substitution.adoc

```diff
@@ -2,21 +2,36 @@
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. Is at least partially inside the <<Field Surface, field margin>>
+.. At a distance from the halfway line that must not exceed 30 centimeters
+.. The ball must be at least 2 meters away from the halfway line
+
+. If the previous specifications are respected, the <<Robot Handler, robot handler>> is allowed to perform the following operations.
+... Put robots in, without notifying the <<Referee, referee>>.
+... Take robots out, without notifying the <<Referee, referee>>.
+
+. Given that robots can be put in at any time, while respecting the previous specifications, on a game stoppage robots can only be removed.
+.. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs the following operation in the given order.
+... Take robots out.
+... The <<Robot Handler, robot handler>> informs the <<Referee, referee>> when done.
+... When both teams finished the robot substitution, the <<Referee, referee>> informs the <<Game Controller Operator, game controller operator>>.
+... The <<Game Controller Operator, game controller operator>> performs a <<Stop, stop>> followed by continuing the game.
+
 . If one of the robots taken out is the keeper, the <<Robot Handler, robot handler>> tells the <<Game Controller Operator, game controller operator>> the id of the robot that takes over the keeper role.
-. The <<Robot Handler, robot handler>> informs the <<Referee, referee>> when done.
-. When both teams finished the robot substitution, the <<Referee, referee>> informs the <<Game Controller Operator, game controller operator>>.
-. The <<Game Controller Operator, game controller operator>> performs a <<Stop, stop>> followed by continuing the game.
+
+. The maximum allowed number of robots of the team on the field must not be exceeded.
+
 
 .Usage
 Robots can be substituted for any reason. There is no limit on the number of substitutions.
 
 A robot substitution intent can be made by:
 
 . A <<Robot Handler, robot handler>> by informing the <<Game Controller Operator, game controller operator>> who in turn enters the intent into the <<Game Controller, game controller>>.
-. A team software by sending a request to the <<Game Controller, game controller>>.
```

Comment:
Why is this removed?

Reply:
With the new automatic procedure I didn't see why this would be necessary. Should it be kept?

Reply:
if a robot is broken and must be taken off the field, you still want to request substitution.